en/1-design-with-users.md

```diff
@@ -64,44 +62,8 @@ Users should be involved throughout the lifecycle of the service, with user rese
 
 - Use qualitative and quantitative data to help improve your understanding of user needs and identify areas for improvement **(OneGC product design criteria)**
 
-- Research what data is available **(Think - Digital Design Playbook (ISED))**

```

Comment:
This stuff shouldn't be removed. It is new content that was added by a previous pull request.